src/test/java/com/synopsys/integration/jenkins/polaris/extensions/pipeline/ExecutePolarisCliStepTest.java

```diff
@@ -0,0 +1,85 @@
+package com.synopsys.integration.jenkins.polaris.extensions.pipeline;
+
+import java.io.File;
+import java.util.Optional;
+
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.synopsys.integration.jenkins.polaris.extensions.tools.PolarisCli;
+import com.synopsys.integration.stepworkflow.StepWorkflow;
+
+import hudson.EnvVars;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Node;
+import hudson.model.TaskListener;
+import hudson.remoting.VirtualChannel;
+import com.synopsys.integration.stepworkflow.SubStep;
+import com.synopsys.integration.stepworkflow.StepWorkflowResponse;
+
+@PowerMockIgnore({"javax.crypto.*", "javax.net.ssl.*" })
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({PolarisCli.class, StepWorkflow.class})
+public class ExecutePolarisCliStepTest {
+    private static final String WORKSPACE_REL_PATH = "out/test/PolarisBuildStepTest/testPerform/workspace";
+    private static final String TEST_POLARIS_CLI_NAME = "testPolarisCliName";
+    private static final String TEST_POLARIS_HOME = "/tmp/polaris";
+    public static final String TEST_POLARIS_ARGS = "testArgs";
+
+    // TODO: To improve this test (to test more of ExecutePolarisCliStep.Execution.run(), we could refactor ExecutePolarisCliStep to:
```

Comment:
Is this TODO done?

Reply:
Not yet, but the two test TODOs are basically next. I think these changes will help.